dask/dataframe/tests/test_groupby.py

```diff
@@ -296,6 +296,11 @@ def test_groupby_on_index(scheduler):
     pdf2 = pdf.set_index("a")
     assert_eq(ddf.groupby("a").b.mean(), ddf2.groupby(ddf2.index).b.mean())
 
+    # Check column projection for `groupby().agg`
+    agg = ddf.groupby("a").agg({"b": "mean"})
+    assert_eq(ddf.groupby("a").b.mean(), agg.b)
+    assert hlg_layer(agg.dask, "getitem")
```

Comment:
This passes on `main` -- is this maybe supposed to use `ddf2`?

Reply:
Oops - good catch!